test/conditionSpec.js

```diff
@@ -66,19 +67,19 @@ describe('Conditional transfers with Nerd and Noob', function () {
           assert(transfer.ledger === 'test.nerd.')
           resolve()
         })
-      })
+      }).catch(done)
 
       nerd.send({
         id: 'first',
         account: 'x',
         amount: '100',
         executionCondition: condition
-      })
+      }).catch(done)
 
       return p
     }).then(() => {
       done()
-    })
+    }).catch(done)
```

Comment:
You can just return a promise from a mocha test instead of using `done`, might be cleaner/easier for a lot of these tests.